src/extra/egui/context.cpp

```diff
@@ -0,0 +1 @@
+#include "context.cpp"
```

Comment:
Infinite import loop? I'm surprised it actually compiles, but probably because the file is not referenced in the corresponding `CMakeLists.txt`.

```suggestion
#include "context.hpp"
```

Reply:
Yeah should be the header

Reply:
Actually removed